### DISCUSSION TITLE
Linux: Updating/Upgrading Homebrew Packages Using KDE Discover?

### DISCUSSION THREAD

**Discussion by FeryET** [Homebrew/discussions#4716] [Linux]
### Additional Info

- [ ] I am using Windows Subsystem for Linux

### Output of `brew config`

```shell
HOMEBREW_VERSION: 4.1.4
ORIGIN: https://github.com/Homebrew/brew
HEAD: 98f1be77f93a0d23fbc68629d0854522e9f0c1c3
Last commit: 6 days ago
Core tap JSON: 13 Aug 09:29 UTC
HOMEBREW_PREFIX: /home/linuxbrew/.linuxbrew
HOMEBREW_CASK_OPTS: []
HOMEBREW_DISPLAY: :1
HOMEBREW_MAKE_JOBS: 8
Homebrew Ruby: 2.6.10 => /home/linuxbrew/.linuxbrew/Homebrew/Library/Homebrew/vendor/portable-ruby/2.6.10_1/bin/ruby
CPU: octa-core 64-bit skylake
Clang: N/A
Git: 2.39.2 => /bin/git
Curl: 7.88.1 => /bin/curl
Kernel: Linux 6.2.0-27-generic x86_64 GNU/Linux
OS: Ubuntu 23.04 (lunar)
Host glibc: 2.37
/usr/bin/gcc: 12.3.0
/usr/bin/ruby: N/A
glibc: N/A
gcc@11: N/A
gcc: N/A
xorg: N/A
```

### Output of `brew doctor`

```shell
Your system is ready to brew.
```

### Description of issue

I use KDE Discover to manage the updating/upgrading of installed packages on my system. It supports Flatpaks, Snaps and Debian packages. Is there anyway to integrate Brew with KDE Discover? If not, what should happen to make KDE Discover support Brew like the other package managers?

**Comment by SMillerDev** [comment]
> If not, what should happen to make KDE Discover support Brew like the other package managers?

That sounds like a very good question for the developers of KDE.
